maxim/logger/groq/client.py

```
@@ -0,0 +1,150 @@
+
+_INSTRUMENTED = False
+
+def instrument_groq(logger: Logger) -> None:
+    """Patch Groq's chat completion methods for logging."""
+
+    global _INSTRUMENTED
+    if _INSTRUMENTED:
+        scribe().debug("[MaximSDK] Groq already instrumented")
+        return
+
+    def wrap_sync_create(create_func):
```

Ruff (0.11.9)

Missing return type annotation for private function `wrap_sync_create`

(ANN202)